connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSourceTask.java

```diff
@@ -124,6 +125,10 @@ protected void close() {
     }
 
     @Override
+    protected void releaseResources() {
```

Comment:
Can we just drop this since neither implementation contains any code?

Reply:
Actually, this will be used in the future PRs to close the subclass-specific metrics. I debated about doing this inside close, but the problem is that we'd miss any failures that happen when the tasks are being closed.